examples/amp-story/bookendv1.json

```diff
@@ -15,6 +15,20 @@
       "title": "This is an example article",
       "url": "http://example.com/article.html",
       "image": "http://placehold.it/256x128"
+    },
+    {
+      "type": "cta-link-pair",
+      "text1": "sign up",
```

Comment:
I don't love `text1/url1` and `text2/url2`; this might be partially tied to my other point regarding the spacing but maybe we should just accept an array of `text/url` pairs?  We can then add runtime validation to enforce a maximum if we like.  WDY (and @gmajoulet) T?